Gems/Camera/Code/Source/CameraComponentController.cpp

```diff
@@ -433,6 +449,11 @@ namespace Camera
         CameraNotificationBus::Broadcast(&CameraNotificationBus::Events::OnActiveViewChanged, m_entityId);
     }
 
+    bool CameraComponentController::IsActiveView()
```

Comment:
question: Could this be made `const`?

Reply:
None of the other CameraComponentRequests getter methods (which could potentially be const) are const.

I think this is part of a broader design question about when EBus methods should be const, if ever. In the past, I've rarely seen them marked const.

Reply:
Ah sorry, I think the reason they're not is historically they actually couldn't be (a very long time ago before the more modern C++ 11 machinery was added) and the other is in theory we don't want to limit an implementer of an API about how they implement something and making something const as a restriction might be too onerous.. when calling through an EBus interface you don't care if the function is const or not so it also doesn't make any different there... Personally I'd rather we start adding const for simple getters (in future) and then implementers can still use `mutable` if they need to update a cached value for example... (this is fine to stay as-is for now though thanks! :+1:)